src/components/activities/ActivityCard.jsx

```diff
@@ -1,6 +1,12 @@
 import React, { Component } from 'react';
 import PropType from 'prop-types';
 
+/**
+ * Renders an activity card
+ *
+ * @class ActivityCard
+ * @extends React.Component
+*/
 export default class ActivityCard extends Component {
   state = {};
   statuses = ['pending', 'expired', 'approved', 'default'];
```

Comment:
this variable could be better named to status

Reply:
Yes. The context for his issue is comments.

Reply:
ok